enhancements/api-review/custom-tags-aws.md

```diff
@@ -110,52 +176,97 @@ If a customer decides they do not want these tags applied to new resources, ther
 
 In the future we will want to introduce a spec field where a customer can specify and edit these tags, which will be reflected into status when changed, and we will expect consuming components to reconcile changes to the set of tags by applying net new tags to existing resources (but they will still not remove tags that are dropped from the list of tags).
 
+## Alternatives
+
+Proposal for managing user-defined tags in day2 operations is discussed in the following sections.
+
+## What motivates
+
+Motivations include but are not limited to:
+
+- Allow admin, compliance, and security teams to keep track of assets and objects created by OpenShift,
+both at install time and during continuous operation (Day 2)
+- In a Managed OpenShift environment such as Red Hat OpenShift on AWS (ROSA), allow easy differentiation
+between Red Hat-managed AWS objects and customer-managed objects
+- *Allow for the restriction of permissions granted to Red Hat in an AWS account by AWS resource tags.
+For example, see https://issues.redhat.com/browse/SDE-1146 - "IAM users and roles can only operate on resources with specific tags"*
+
+### Identified goal
+
+- The administrator or service (in the case of Managed OpenShift) installing OpenShift can pass an arbitrary
+list of user-defined tags to the OpenShift Installer, and everything created by the installer and all other
+bootstrapped components will apply those tags to all resources created in AWS, for the life of the cluster, and where supported by AWS.
```

Comment:
Later on we say that this arbitrary list is limited, at least currently, to 10 items. I wonder if that should be mentioned here or if that's just the perils of not reading the entire document.